tests/integration/pillar/test_git_pillar.py

```diff
@@ -106,6 +106,10 @@
 HAS_NGINX = bool(salt.utils.path.which("nginx"))
 HAS_VIRTUALENV = bool(salt.utils.path.which_bin(VIRTUALENV_NAMES))
 
+pytestmark = [
```

Comment:
I don't think we want to skip the whole file, just the SSH pygit2 tests.  Or if we want all the pygit2 tests.  We shouldn't skip the GitPython tests.

And we use the same version of libgit2-dev on x86_64, so it's not that it's outdated, it's just buggy.